### PR TITLE
Choice graphic variant

### PR DESCRIPTION
**Pull request by philschanely**
## Description

Modifies the Choice component to allow for a graphic variation and a new link-text slot on the right side.

### Screenshots

Screen Shot 2021-01-12 at 4 10 11 PM

### Testing

See http://localhost:4000/pages/element/choice and the new `Graphic Variation (with link text)` block.

### CODE REVIEW

File: packages/sage-react/lib/Tabs/TabsItem.jsx [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
   itemStyle,
+  linkText,
   onClick,
   panelId,
   subtext,
   type,
   ...rest
 }) => {
+  const { to, href } = rest;
+  console.log(to, href);
```
Comment (joedakroub) [inline packages/sage-react/lib/Tabs/TabsItem.jsx:25]: Just checking, is this `console.log` intentional?

**Review by joedakroub — APPROVED** [review]
Looks great locally. Added one comment. Nice addition! 👍

File: packages/sage-react/lib/Tabs/TabsItem.jsx (hunk 1 above)
Comment (philschanely) [reply packages/sage-react/lib/Tabs/TabsItem.jsx:25]: 😲

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼  Great work @philschanely! 🔥

**Pull request merged**